test/binding.gyp

```diff
@@ -27,6 +27,7 @@
         'error.cc',
         'external.cc',
         'function.cc',
+        'funcRefObject.cc',
```

Comment:
Please make sure the file names and the name of the init function are consistent for https://github.com/nodejs/node-addon-api/pull/1078 by @deepakrkris!

Reply:
@deepakrkris this is interesting, because it creates a situation where a test file has a dependency. Maybe this class definition should go to the top of function_reference.cc instead because that's the only file using it. Then we can also avoid having to deal with file dependencies.